front/lib/api/workspace.ts

```diff
@@ -80,6 +80,20 @@ export async function getWorkspaceVerifiedDomain(
   return null;
 }
 
+export async function removeAllWorkspaceDomains(
+  workspace: LightWorkspaceType
+): Promise<void> {
+  const workspaceDomains = await WorkspaceHasDomainModel.findAll({
+    where: {
+      workspaceId: workspace.id,
+    },
+  });
+
+  for (const domain of workspaceDomains) {
+    await domain.destroy();
+  }
```

Comment:
Good for me, but maybe we can do it faster? Probably limited has there isn't a lot of workspace_has_domain attached.

```suggestion
  await WorkspaceHasDomainModel.destroy({
    where: {
      workspaceId: workspace.id,
    },
  });
```